irc/weechat/Portfile

```diff
@@ -78,32 +77,37 @@ configure.args-append \
 
 variant python requires python27 description {Compatibility variant, requires +python27} {}
 
-variant python27 description "Bindings for Python 2.7 plugins" conflicts python36 python37 python38 python39 {
+variant python27 description "Bindings for Python 2.7 plugins" conflicts python36 python37 python38 python39 python310 {
     configure.args-replace  -DENABLE_PYTHON=OFF -DENABLE_PYTHON=ON
     configure.args-replace  -DENABLE_PYTHON2=OFF -DENABLE_PYTHON2=ON
     depends_lib-append      port:python27
 }
 
-variant python36 description "Bindings for Python 3.6 plugins" conflicts python27 python37 python38 python39 {
+variant python36 description "Bindings for Python 3.6 plugins" conflicts python27 python37 python38 python39 python310 {
     configure.args-replace  -DENABLE_PYTHON=OFF -DENABLE_PYTHON=ON
     depends_lib-append      port:python36
 }
 
-variant python37 description "Bindings for Python 3.7 plugins" conflicts python27 python36 python38 python39 {
+variant python37 description "Bindings for Python 3.7 plugins" conflicts python27 python36 python38 python39 python310 {
     configure.args-replace  -DENABLE_PYTHON=OFF -DENABLE_PYTHON=ON
     depends_lib-append      port:python37
 }
 
-variant python38 description "Bindings for Python 3.8 plugins" conflicts python27 python36 python37 python39 {
+variant python38 description "Bindings for Python 3.8 plugins" conflicts python27 python36 python37 python39 python310 {
     configure.args-replace  -DENABLE_PYTHON=OFF -DENABLE_PYTHON=ON
     depends_lib-append      port:python38
 }
 
-variant python39 description "Bindings for Python 3.9 plugins" conflicts python27 python36 python37 python38 {
+variant python39 description "Bindings for Python 3.9 plugins" conflicts python27 python36 python37 python38 python310 {
     configure.args-replace  -DENABLE_PYTHON=OFF -DENABLE_PYTHON=ON
     depends_lib-append      port:python39
 }
 
+variant python310 description "Bindings for Python 3.10 plugins" conflicts python27 python36 python37 python38 python310 {
```

Comment:
this cannot conflict with `python310`, that should be `python39`

Reply:
Good catch! Fixed.